lib/azure/resource_management/ARM_interface.rb

```diff
@@ -139,9 +139,8 @@ def delete_server(resource_group_name, vm_name)
           ui.info 'Deleting ..'
 
           begin
-            print '.'
-            promise = compute_management_client.virtual_machines.delete(resource_group_name, vm_name)
-          end until promise.value!.body.nil?
+            server_detail = compute_management_client.virtual_machines.delete(resource_group_name, vm_name)
+          end until server_detail.value!.body.nil?

```

Comment:
@NimishaS :- For this api call, we are getting `.value!.body` in `server_detail` in later version of `azure-sdk-for-ruby`.